site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -343,13 +340,13 @@ Here are the default items and their Ids:
 
 include::docs/v1/tech/shared/_default-entities.adoc[]
 
-With the known Id, you can build other configuration in your Kickstart file that depend on or copy them. For example, you may copy the default theme and modify the copied theme via Kicstart, or you may create a new tenant but set the tenant Id token signing key to one of the shadow keys.
+With the known Id, you can build other fields in your Kickstart file that depend on or copy them. For example, you may copy the default theme and modify the copied theme via Kickstart, or you may create a new tenant but set the tenant Id token signing key to one of the shadow keys.
```

Comment:
The first sentence seems off to me but I am not sure why.  After reading several times, it makes sense, but I shouldn't have to re-read to understand.  This is just a suggestion.
```suggestion
With the known Ids, you can build other fields in your Kickstart file that utilizes them. For example, you may copy the default theme and modify the copied theme via Kickstart, or you may create a new tenant but set the tenant Id token signing key to one of the shadow keys.
```